### PR TITLE
JavaScript Capstone Project Final Review #37

### PR DESCRIPTION
**Pull request by bobb-Rob**
## We (team of two) Implemented the following:

1.    When the page loads, the webapp retrieves data from:

      - The selected API shows the list of items on the screen.
      - The Involvement API to show the item likes.

3.   Our page makes only 2 requests:
     - One to the base API.
     - And one to the Involvement API.
4.   When the user clicks on the Like button of an item, the interaction is recorded in the Involvement API and the screen is updated.
5.   When the user clicks on the "Comments" button, the Comments popup appears.
      - When the popup loads, the webapp retrieves data from:
              - The selected API and shows details about the selected item.
              - The Involvement API to show the item comments.
      - When the user clicks on the "Comment" button, the data is recorded in the Involvement API and the screen is updated.
7.   Home page header and navigation similar to the given mockup.
8.   Home page footer similar to the given mockup.
9.   Counters We have counters in all the interfaces that show:
      - The number of items (home).
      - The number of comments (comments popup).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,56 +1,80 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Movie Rating Website
 
-> Description the project.
+> This is a website that consumes TV maze API and allows users to like and post reviews of movies they have watched. It is a capstone project for JavaScript and Networking Module two of the microverse curricullum.
 
+## [Watch a video description of the features](https://drive.google.com/file/d/1ysVUkkB0vx2lSc_-8Gol2e5RUtHi2y5R/view?usp=sharing)
```
Comment (topeogunleye) [inline README.md:7]: - [ ] Nice job creating a demo video about your project features, however, please also add the following to your video:

- All the members in the video should talk about the project i.e your partner.
- The video should contain a highlight about a piece of code that you find interesting or are proud of making.

**Review by topeogunleye — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi Team,
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Gitflow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
> * [ ]  Please check my inline comments.
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @topeogunleye in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/styles/style.css [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+  margin: 0 auto;
+}
+
+.card-img-top :hover {
+  transform: scale(1.5);
+}
+
+/* .navbar {
+  margin: 24px auto;
+} */
```
Comment (CollinsTatang) [inline src/styles/style.css:44]: ### **Optional**

- [ ] Please remove all commented code from your codebase to make your code clean and easy to maintain

File: src/styles/style.css [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+.like-button {
+  cursor: pointer;
+}
+
+/* .card-img-top {
+  width: 100%;
+  height: 30vw;
+  object-fit: cover;
+} */
```
Comment (CollinsTatang) [inline src/styles/style.css:25]: ### **Optional**

- [ ] Please remove all commented code from your codebase to make your code clean and easy to maintain

**Review by CollinsTatang — APPROVED** [review]
Hi @bobb-Rob and @ndikumanaisaie , 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

image

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: src/styles/style.css (hunk 3 above)
Comment (bobb-Rob) [reply src/styles/style.css:25]: Thank you very much for this review. I will get it sorted out

File: src/styles/style.css (hunk 2 above)
Comment (bobb-Rob) [reply src/styles/style.css:44]: All commented code have been removed from the code base

**Pull request merged**